scripts/git_changes_utils.py

```diff
@@ -55,41 +55,30 @@ def get_local_git_repository_remote_name() -> Optional[bytes]:
                 stderr=subprocess.PIPE)
             remote_url, err = task.communicate()
             if not err:
-                if remote_url.endswith(b'oppia/oppia.git\n'):
+                if (
+                    remote_url.endswith(b'oppia.git\n') and
+                    not remote_url.endswith(b'oppia/oppia.git\n')
+                ):
                     remote_num += 1
                     remote_name = remote
             else:
                 raise ValueError(err)
     else:
         raise ValueError(err)
 
-    if not remote_num:
+    if remote_num == 0:
         raise Exception(
-            'Error: Please set the git \'upstream\' repository.\n'
+            'Error: Please set the git \'origin\' repository.\n'
             'To do that follow these steps:\n'
             '1. Run the command \'git remote -v\'\n'
-            '2a. If \'upstream\' is listed in the command output, then run the '
-            'command \'git remote set-url upstream '
-            'https://github.com/oppia/oppia.git\'\n'
-            '2b. If \'upstream\' is not listed in the command output, then run '
-            'the command \'git remote add upstream '
-            'https://github.com/oppia/oppia.git\'\n'
+            '2a. If \'origin\' is listed in the command output, then run the '
+            'command \'git remote set-url origin '
+            '\"The URL of your fork of Oppia GitHub repository\"\'\n'
+            '2b. If \'origin\' is not listed in the command output, then run '
+            'the command \'git remote add origin '
+            '\"The URL of your fork of Oppia GitHub repository\"\'\n'
         )
 
-    if remote_num > 1:
```

Comment:
Should we still have an error if > 1, to avoid ambiguity?

Reply:
Oh, hm, I'm not sure I've ever seen that case come up. I feel a bit like we should be restrictive here for now, and then if issues arise let's figure out what the use case is and modify accordingly, since the alternative might end up just doing the wrong thing here silently (right?) -- since devs might not read stuff that's printed in the middle of the log.

Reply:
Sorry, so should we change it to raise an Exception if the remote num is over 1 then?